src/RequestInsurance/ViewComponents/PrettyPrintDifference.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Cego\RequestInsurance\ViewComponents;
+
+use Exception;
+use Illuminate\View\View;
+use Jfcherng\Diff\Differ;
+use Jfcherng\Diff\DiffHelper;
+use Illuminate\View\Component;
+use Jfcherng\Diff\Factory\RendererFactory;
+
+class PrettyPrintDifference extends Component
+{
+    /**
+     * String with HTML displaying the difference between old string and new string.
+     *
+     * @var string
+     */
+    public $content;
+
+    /**
+     * Rendering options used when creating string of HTML.
+     *
+     * @var array
+     */
+    protected array $rendererOptions = [
+        'detailLevel'         => 'word',
+        'showHeader'          => true,
+        'separateBlock'       => true,
+        'resultForIdenticals' => null,
+        'lineNumbers'         => false,
+    ];
+
+    /**
+     * Additional options used when looking for differences in the two given strings.
+     *
+     * @var array
+     */
+    protected array $differOptions = [
+        'ignoreWhitespace' => false,
+        'context'          => 3,
+    ];
+
+    /**
+     * Create a new component instance.
+     *
+     * @param object|array|string $content
+     *
+     * @throws \JsonException
+     */
+    public function __construct($oldValues, $newValues)
+    {
+        $this->content = $this->prettyPrint($oldValues, $newValues);
+    }
+
+    /**
+     * Get the view / contents that represent the component.
+     *
+     * @return \Illuminate\Contracts\View\Factory|View
+     */
+    public function render()
+    {
+        return view('request-insurance::components.pretty-print-difference');
```

Comment:
Du behøver ikke et view, når det eneste dit view render er en streng. Så kan du bare return $this->content her :smile:

Reply:
Og slette dit view selvfølelig.